src/layout/footer.html

```diff
@@ -29,67 +29,73 @@ <h3 class="column_name">
             Company
         </h3>
         
-            <div class="column_name_list">
-                <a class="column_name_list_link" href="#">
-                     About
-                </a>
-            </div>
-            <div class="column_name_list">
-                <a class="column_name_list_link"  href="#">
-                     Career
-                </a>
-            </div>
-            <div class="column_name_list">
-                <a class="column_name_list_link" href="#">
-                     Mobile
-                </a>
-            </div>
+           <div class="list_name">
+                <div class="column_name_list">
+                    <a class="column_name_list_link" href="#">
+                         About
+                    </a>
+                </div>
+                <div class="column_name_list">
+                    <a class="column_name_list_link"  href="#">
+                         Career
+                    </a>
+                </div>
+                <div class="column_name_list">
+                    <a class="column_name_list_link" href="#">
+                         Mobile
+                    </a>
+                </div>
+           </div>
         
     </div>
     <div class="column">
         <h3 class="column_name">
             Contact
         </h3>
         
-            <div class="column_name_list">
-                <a class="column_name_list_link" href="#">
-                     Why Travlog?
-                </a>
-            </div>
-            <div class="column_name_list">
-                <a class="column_name_list_link" href="#">
-                     Partner with us
-                </a>
-            </div>
-            <div class="column_name_list">
-                <a class="column_name_list_link" href="#">
-                     FAQ's
-                </a>
-            </div>
-            <div class="column_name_list">
-                <a class="column_name_list_link" href="#">
-                     Blog
-                </a>
+            <div class="list_name">
+                <div class="column_name_list">
+                    <a class="column_name_list_link" href="#">
+                         Why Travlog?
+                    </a>
+                </div>
+                <div class="column_name_list">
+                    <a class="column_name_list_link" href="#">
+                         Partner with us
+                    </a>
+                </div>
+                <div class="column_name_list">
+                    <a class="column_name_list_link" href="#">
+                         FAQ's
+                    </a>
+                </div>
+                <div class="column_name_list">
+                    <a class="column_name_list_link" href="#">
+                         Blog
+                    </a>
+                </div>
```

Comment:
?

Reply:
I don't know how to create components by html for it

Reply:
![image](https://github.com/user-attachments/assets/c37a388b-b1c1-4261-8e56-2d38d4b1df31)
split to component like this and reuse like this:
![image](https://github.com/user-attachments/assets/61e096b1-495f-4520-a52b-bdc07e9e766e)

Reply:
Thank you. I will try it after . when i finished all responsive